audio/midi_drivers/FluidSynthMidiDriver.cpp

```diff
@@ -109,9 +122,12 @@ int FluidSynthMidiDriver::open() {
 
 void FluidSynthMidiDriver::close() {
 	while (!_soundFont.empty()) {
-		int soundfont = _soundFont.top();
+		//-- Do not sfunload the SoundFonts, this causes messages :
+		//--   fluidsynth: warning: No preset found on channel # [bank=# prog=#]
+		//-- And delete_fluid_synth unloads the SoundFonts anyway.
+		// int soundfont = _soundFont.top();
 		_soundFont.pop();
-		fluid_synth_sfunload(_synth, soundfont, 1);
+		// fluid_synth_sfunload(_synth, soundfont, 1);
 	}
```

Comment:
Might as well replace the whole loop by `_soundFont.clear();`

Reply:
Sorry, cannot. _soundFont is a std::stack, not a std::vector. It has no clear method.